projects/ngx-sub-form/src/lib/deprecated/ngx-sub-form.component.ts

```diff
@@ -173,9 +173,20 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
             }
           }
 
+          let value = undefined;
+          if (control && filterControl(control, key, false)) {
+            value = {
+              ...mapControl(control, key)
+            };
+          }
+
           if (values.length > 0 && values.some(x => !isNullOrUndefined(x))) {
-            controls[key] = values;
+            value = {
+              ...value,
+              ...values
```

Comment:
I didn't spend to much investigation into the old implementation because they will be removed. This works into my fork and project. I think we should not care about this.